src/www/ui/api/Controllers/UploadController.php

```diff
@@ -417,11 +417,11 @@ private function changeUpload($request, $response, $args, $isCopy)
    */
   public function postUpload($request, $response, $args)
   {
-    $uploadType = $request->getHeaderLine('uploadType');
+    $reqBody = $this->getParsedBody($request);
+    $uploadType = $reqBody['uploadType'] ?? null;
```

Comment:
The API should still support V1 requests in deprecated mode until it is completely removed. You can read the version of request with `ApiVersion::getVersion($request) == ApiVersion::V2`.

See https://github.com/fossology/fossology/pull/2617/files#diff-1bae1ab555e908fff79a5b02ff0f05677a8d1f5653259e98531071acfeb7e16bR42 for reference.